specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2023-09-01-preview/examples/ApiManagementDeleteGatewayConfigConnection.json

```diff
@@ -0,0 +1,30 @@
+{
+  "parameters": {
+    "resourceGroupName": "rg1",
+    "api-version": "2023-09-01-preview",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "gatewayName": "standard-gw-01",
+    "configConnectionId": "gcc-01",
+    "If-Match": "*"
+  },
+  "responses": {
+    "202": {
+      "headers": {
+        "location": "https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg1/providers/Microsoft.ApiManagement/gateways/standard-gw-1/configConnections/gcc-01?api-version=2023-09-01-preview",
+        "Azure-AsyncOperation": "https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg1/providers/Microsoft.ApiManagement/gateways/standard-gw-1/configConnections/gcc-01?api-version=2023-09-01-preview"
+      },
+      "body": {
+        "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg1/providers/Microsoft.ApiManagement/gateways/standard-gw-1/configConnections/gcc-01",
+        "name": "gcc-01",
+        "type": "Microsoft.ApiManagement/gateways/configConnections",
+        "etag": "AAAAAAAWN/4=",
+        "properties": {
+          "provisioningState": "Succeeded",
+          "sourceId": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg1/providers/Microsoft.ApiManagement/services/apim-service-1/workspaces/ws-001",
+          "defaultHostname": "gcc-01-amf2h5hpf7gafbeu.standard-gw-1.gateway.eastus.azure-api.net"
+        }
+      }
```

Comment:
[ARMBlockingComment] I don't think 202 is supposed to have a response body containing the resource like this.
Keep in mind that /delete RBAC permission doesn't imply /read RBAC permission.

Reply:
Got it I can remove that